.internal-ci/helm/fog-services/templates/fog-report-grpc-ingress.yaml

```diff
@@ -13,6 +13,9 @@ metadata:
     {{- if $.Values.global.certManagerClusterIssuer }}
     cert-manager.io/cluster-issuer: {{ $.Values.global.certManagerClusterIssuer }}
     {{- end }}
+    {{- if (include "fogServices.blocklist.enabled" $) }}
+    haproxy.org/blacklist: {{ include "fogServices.blocklist.pattern" $ }}
```

Comment:
I notice most other templates pass `.`, while these `fog-report*` templates use `$`. What's the difference?

Reply:
See nick's question above :) TL;DR this template is in a loop so to access the outer context you need to use `$`